utils.sh

```diff
@@ -146,13 +153,6 @@ function master_node_to_install_config() {
 EOF
 }
 
-function collect_info_on_failure() {
-    $SSH -o ConnectionAttempts=500 core@$IP sudo journalctl -b -u bootkube
-    oc get clusterversion/version
-    oc get clusteroperators
-    oc get pods --all-namespaces | grep -v Running | grep -v Completed
-}
```

Comment:
Can you remove the call to this in https://github.com/openshift-metalkube/dev-scripts/blob/master/06_create_cluster.sh#L36? I don't think it should be there

Reply:
(Or restore this, if we do want to collect those logs)

Reply:
Done

Reply:
This is no longer needed - it was an attempt to collect logs when install fails. It didn't work and got properly implemented in `run-ci`